src/main/java/aws/retrospective/entity/Retrospective.java

```diff
@@ -98,8 +98,8 @@ public void update(String title, ProjectStatus status, UUID thumbnail, String de
         this.description = description;
     }
 
-    public boolean isSameTemplate(TemplateSection templateSection) {
-        return this.getTemplate().getName().equals(templateSection.getTemplate().getName());
+    public boolean isNotSameTemplate(TemplateSection templateSection) {
+        return !this.getTemplate().getName().equals(templateSection.getTemplate().getName());
     }
```

Comment:
```suggestion
public boolean isNotSameTemplate(RetrospectiveTemplate template) {
    return !this.getTemplate().getName().equals(template.getName());
}
```
Template이 같은지 확인하는 메서드이니 TemplateSection에 있는 Template을 꺼내서 비교하는게 더 좋을 것 같음!

Reply:
수정하니까 코드가 잘 읽히네요👍👍